src/browser/api/webcontents.ts

```diff
@@ -19,11 +19,7 @@ export function hookWebContentsEvents(webContents: Electron.WebContents, { uuid,
     ) => {
         const type = 'resource-response-received';
 
-        const payload = {
-            name,
-            uuid,
-            topic,
-            type,
+        const payload = { uuid, name, topic, type,
```

Comment:
:neckbeard: why single line ?

Reply:
Lol it's the only way to pass the linter. 100 line limit.